packages/shared/src/components/search/SearchPanel/SearchPanel.tsx

```diff
@@ -144,7 +144,7 @@ export const SearchPanel = ({ className }: SearchPanelProps): ReactElement => {
       >
         <SearchPanelInput
           className={{
-            container: classNames('w-full laptop:w-[35rem]'),
+            container: classNames('w-full laptop:max-w-[35rem]'),
```

Comment:
Isn't this supposed to be a fixed width?

Reply:
If it's fixed width, the navbar overflows on smallest laptop screens with the starwars logo. Because the starwars logo is wider.

Reply:
I'm just afraid max-w will introduce some issues, pretty sure we had a issue with this exact line not long ago 😅

Reply:
Not sure what else can be done here, other than changing the size to be smaller?

Reply:
Maybe we can change the size to something smaller if the feature is on on smaller screens?

Reply:
Did just that, on laptop breakpoint new width is set to 29.5 rem (the max width until the layout starts overflowing) and on laptopL I kept the original width of 35 rem.